.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -5,36 +5,53 @@ about: Create a report to help us improve
 
 <!-- Before you open a new issue, search through the existing issues to see if others have had the same problem.
 
+DO NOT DELETE ANYTHING FROM THIS TEMPLATE.
+
 Issues not containing the minimum requirements will be closed:
 
 - Issues without a description (using the header is not good enough) will be closed.
+- Issues without a Google version wil be closed
```

Comment:
I don't like this threatening language. How about something more kind?

Reply:
I just reused what was already there. Will come up with a better phrasing 😄